src/desktop/apps/artwork/components/commercial/templates/acquire.jade

```diff
@@ -1,4 +1,6 @@
-if artwork.is_acquireable
+- var newBuyNowFlow = user && user.hasLabFeature("New Buy Now Flow")
+- var auctionPartner = artwork.partner.type === 'Auction' || artwork.partner.type === 'Auction House'
```

Comment:
There _are_ some stranded partner-less artworks around, we should possibly be defensive about that here (`artwork.partner.type` would crash on those).